service/src/main/java/uk/gov/hmcts/reform/fpl/service/validators/RespondentsChecker.java

```diff
@@ -24,17 +23,14 @@
 @RequiredArgsConstructor(onConstructor_ = {@Autowired})
 public class RespondentsChecker extends PropertiesChecker {
     private final FeatureToggleService featureToggleService;
+    private final RespondentAfterSubmissionValidator respondentAfterSubmissionValidator;
 
     @Override
     public List<String> validate(CaseData caseData) {
 
-        List<Class<?>> groups = new ArrayList<>();
-        groups.add(Default.class);
-
-        if (featureToggleService.isRespondentJourneyEnabled()) {
```

Comment:
nice bit of clean up, if you want you could remove the blank line at the start of this method

Reply:
also could clean up the other useage of the feature toggle as well and remove the static from the private method

Reply:
will cleanup feature toggle in another pr